backend/tsconfig.json

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
Moved to `packages/backend/src/tsconfig.json`. GitHub didn't pick it due to some changes introduced.